portmgr/dmg/postflight.in

```diff
@@ -278,8 +279,8 @@ if /usr/bin/su "${USER}" -l -c "/usr/bin/printenv MANPATH" > /dev/null; then
     fi
 fi
 
-# Adding a DISPLAY variable only if we're running on Tiger or less and if it doesn't already exist:
-if (($(sw_vers -productVersion | awk -F . '{print $2}') >= 5)) || /usr/bin/su "${USER}" -l -c "/usr/bin/printenv DISPLAY" > /dev/null > /dev/null; then
+# Adding a DISPLAY variable only if we're running on Tiger (Darwin8) or less and if it doesn't already exist:
+if (($(uname -r | awk -F . '{print $1}') >= 9)) || /usr/bin/su "${USER}" -l -c "/usr/bin/printenv DISPLAY" > /dev/null > /dev/null; then
```

Comment:
Do we need to refer to `awk` by its full path or not? We appear to be inconsistent on this, even within this file. Perhaps we are doing this to ensure that we do not find a GNU awk that might be incompatible with our BSD awk syntax, although this syntax is pretty simple and works with either. But `cut -d. -f1` would be even simpler.

Reply:
I noticed that but figured that was not relevant to the issue here, and I wanted to keep the changes the minimum needed to address the OS version issue. Changes to how the output is parsed should I think be a separate PR